components/FeatureRequestDialog.tsx

```diff
@@ -25,15 +30,22 @@ export function FeatureRequestDialog() {
   const form = useForm<FeatureRequestFormData>();
   const [isSubmitting, setIsSubmitting] = useState(false);
 
+  const submitFeatureRequest = useMutation(api.myFunctions.submitFeatureRequest);
+
   const onSubmit = async (data: FeatureRequestFormData) => {
     setIsSubmitting(true);
     try {
-      // Here you would implement logic to submit the feature request
+      await submitFeatureRequest({
+        title: data.subject,
+        description: data.description,
+      });
       console.log("Feature request submitted:", data);
+      toast.success("Feature request submitted successfully!");
       setOpen(false);
       form.reset();
     } catch (error) {
       console.error("Error submitting feature request:", error);
```

Comment:
Use my logger refererence in 'logger.ts' instead of console.log for logging errors